scripts/run_example.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import argparse
+import math
+import os
+import psutil
+import subprocess
+import sys
+
+try:
+  import termcolor
+  canColor = True
+except ImportError:
+  canColor = False
+
+def logFile(jsonFile):
+  base = os.path.splitext(os.path.basename(jsonFile))[0]
+  return '/tmp/supersimtest_{0}'.format(base)
+
+def good(s):
+  if canColor:
+    print('  ' + termcolor.colored(s, 'green'))
+  else:
+    print('  ' + s)
+
+def bad(s):
+  if canColor:
+    print('  ' + termcolor.colored(s, 'red'))
+  else:
+    print('  ' + s)
+
+def main(args):
+  if args.check:
+    subprocess.check_call('valgrind -h', shell=True,
+                          stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+  # check if binary exists
+  assert os.path.exists('supersim')
+
+  # generate command
+  cmd = './supersim {0}'.format(args.config_file)
```

Comment:
this might be annoying because now you can't run run_example.py except under bazel. One option would be to make a flag for the supersim path, which defaults to bazel-bin/supersim but can be overridden in the py_test

Reply:
Ok, fixed.